pages/documentation/download.html

```diff
@@ -24,6 +24,8 @@ <h2 class="usa-heading">Code</h2>
 
 <!--<span class="link-download-subtext">Version {{ site.uswds_version }}</span>-->
 
+<div class="site-note"><strong>Note:</strong> The file date defaults to October 26, 1985, when you download and install this package. This is intentional. NPM sets a default date to ensure reproducible builds for npm packages.</div>
```

Comment:
This is the re-write Amy L. suggested. I think it's a good improvement:

Note: Files in the downloadable USWDS package will show a "last modified" date of October 26, 1985. This is intentional. Npm sets this default date on all its packages to ensure reproducible builds.

We should update the capitalization of npm at the beginning of sentences elsewhere on this page or rewrite to avoid putting npm at the beginning of sentences. Should we do that here or in another PR?

Reply:
Just dropping in my two cents with a small tweak to Amy's language:

Note: Files in the downloadable USWDS package will show a "last modified" date of October 26, 1985. This is intentional. This default date is set by npm on all its packages to ensure builds can be duplicated.

I moved "npm" in the second sentence so we wouldn't have to capitalize, and I modified how we talk about something being duplicated - "reproducible" is a tougher word to get through. If that's technically inaccurate, we can leave it. It still checks out in Readability app for good reading level.

Reply:
@finekatie what do you think about "ensure builds are identical" instead? From my understanding, I think that's more on the nose in this scenario 🤔

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/2621/commits/a227cd7e061f2d768962946c7816f482b8e52219